test/fixtures_s3_test.go

```diff
@@ -0,0 +1,148 @@
+//+build integration fixtures
+
+package test_test
+
+import (
+	"fmt"
+	"github.com/applike/gosoline/pkg/blob"
+	"github.com/applike/gosoline/pkg/fixtures"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/applike/gosoline/pkg/test"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"testing"
+)
+
+const (
+	BucketName    = "s3-fixtures-test-bucket"
+	BasePath      = "test_data/s3_fixtures_test_data"
+	BasePathPurge = "test_data/s3_fixtures_purge_test_data"
+)
+
+type FixturesS3Suite struct {
+	test.Suite
+
+	loader fixtures.FixtureLoader
+	client s3iface.S3API
+	logger mon.Logger
+	mocks  *test.Mocks
+}
+
+func (s *FixturesS3Suite) SetupSuite() []test.SuiteOption {
+	return []test.SuiteOption{
+		test.WithoutAutoDetectedComponents("localstack"),
+		test.WithLogLevel("debug"),
+		test.WithConfigFile("test_configs/config.s3.test.yml"),
+		test.WithConfigFile("test_configs/config.fixtures_s3.test.yml"),
+	}
+}
+
+func TestFixturesS3Suite(t *testing.T) {
+	s := new(FixturesS3Suite)
+	test.RunSuite(t, s)
+}
+
+func (s FixturesS3Suite) TestS3(app test.AppUnderTest) {
+	// CHECK: Alternative for the affectations below.
+	s.logger = mon.NewLogger()
+	s.loader = fixtures.NewFixtureLoader(s.Env().Config(), s.logger)
+	s.client = blob.ProvideS3Client(s.Env().Config())
```

Comment:
if you implement `SetupSuite()`, then you can return a slice of Options in which you can setup your environment.

Reply:
I already had `SetupSuite()` implemented. `SetupTest()` was used to init `client` and `loader` since a new container is launched for each test.